sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/action/ActionCreationContext.java

```diff
@@ -46,5 +46,8 @@ public interface ActionCreationContext extends Context {
    *
    * @return A tracer for the current action, if tracing is configured.
    */
+  @Deprecated(since = "1.4.2")
   Optional<Tracer> getOpenTelemetryTracer();
+
+  Tracer getTracer();
```

Comment:
```suggestion
/**
   * Get an OpenTelemetry tracer for the current action. This will allow for building and automatic
   * exporting of spans.
   *
   * @return A tracer for the current action, if tracing is configured. Otherwise, a noops tracer.
   */
  Tracer getTracer();
```